lib/Project.js

```diff
@@ -455,6 +455,33 @@ Project.prototype.close = function(cb) {
     var base = this.options.id;
     var extractedPath = path.join(dir, base + "-extracted.xliff");
     var extracted = new TranslationSet(this.sourceLocale);
+    var resourceList = [];
+
+    if (typeof this.options.manifest !== undefined) {
+        logger.info(" Creating manifest file ...");
+        var resourcePath = this.options.manifest["dir"] || ".";
+        var manifestName = this.options.manifest["name"] || "ilibmanifest.json";
+        function walk(resourcePath, internalPath) {
+            var list = fs.readdirSync(path.join(resourcePath, internalPath));
+            list.forEach(function (file) {
```

Comment:
This code will grab all json files in that directory, even if they were already there and not generated by this run of the loctool. Is that what is intended? If it were in the plugin code, you could keep track of which files were generated and only put those into the json file.